agent/api/task/task.go

```diff
@@ -159,6 +159,9 @@ const (
 	serviceConnectContainerMappingEnvVar    = "APPNET_CONTAINER_IP_MAPPING"
 	// ServiceConnectAttachmentType specifies attachment type for service connect
 	serviceConnectAttachmentType = "serviceconnectdetail"
+
+	// EBSVolumeType specifies attachment type for ebs volumes attachments
+	EBSVolumeType = "AmazonElasticBlockStorage"
```

Comment:
Can we use the same const var from the shared lib?

Reply:
done.